.circleci/config.yml

```diff
@@ -208,17 +208,25 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/workspace
+      - checkout
       - run: |
-          cd /tmp/workspace/distribution-scripts
+          case "$CIRCLE_BRANCH" in
+          release/*)
+            # In release branches, always use version from src/VERSION
+            VERSION=$(cat src/VERSION)
+            ;;
+          *)
+            # The version is based on the branch name.
+            VERSION=$CIRCLE_BRANCH
+
+            # We need to sanitize it because there are restrictions on some places
+            # where the version is use (Mac pkg names, snap branch).
+            VERSION=${VERSION//_/-}
+            VERSION=${VERSION//\//-}-dev
```

Comment:
This should be applied unconditionally I guess (to be on the safe side).

Reply:
I figure if `src/VERSION` has underscores or slashes, we're in more trouble anyways. So this isn't technically necessary.